tests/quality/test_shrink_quality.py

```diff
@@ -88,11 +88,13 @@ def test_minimize_3_set_of_tuples():
         lambda x: len(x) >= 2) == set(((0,), (1,)))
 
 
-def test_minimize_sets_of_sets():
+@pytest.mark.parametrize('seed', [11, 43, 49, 56, 65])
```

Comment:
Are these seeds significant?

Reply:
Not super significant. They're ones that previously failed when I ran with seeds of 0 to 100 (I've also run the tests in the new form from 0 to 100 and they pass all of them).

Reply:
If the test now passes for any seed I'd rather not seed at all - just bump down the size.